server/plugin/command.go

```diff
@@ -391,20 +392,36 @@ func (p *Plugin) handleSettings(_ *plugin.Context, _ *model.CommandArgs, paramet
 	}
 
 	setting := parameters[0]
-	if setting != settingNotifications && setting != settingReminders {
+	settingValue := parameters[1]
+
+	switch setting {
+	case settingNotifications:
+		switch {
+		case settingValue == settingOn:
+			userInfo.Settings.Notifications = true
+		case settingValue == settingOff:
+			userInfo.Settings.Notifications = false
```

Comment:
This can be cleaned up by using the `settingValue` in the switch statement

```suggestion
		switch settingValue {
		case settingOn:
			userInfo.Settings.Notifications = true
		case settingOff:
			userInfo.Settings.Notifications = false
```

Reply:
done